dgraph/sample/insert.gql

```diff
@@ -0,0 +1,342 @@
+mutation {
```

Comment:
similar to the issue with `query.sh` - this doesn't work for me!
can you give me a hand to try it out?

Reply:
🤦‍♀️  wasn't just you! It should be fixed with e4c8c29996f4e8c21348da4fe019cf4232a94de2